prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -47,6 +47,44 @@ items:
       - kind: ServiceAccount
         name: weave-net
         namespace: kube-system
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Should `prog/weave-kube/weave-daemonset-k8s-1.7.yaml` and `prog/weave-kube/weave-daemonset.yaml` also be updated?

Reply:
@mikebryant `prog/weave-kube/weave-daemonset-k8s-1.7.yaml` seems like a good bet, and I've added that to the PR. 

The other one, however, seems to be a config file that covers v1.5, which didn't support the RBAC permission system (it used ABAC) and was permissive by default.